### PR TITLE
Updated about first page.

### PR DESCRIPTION
**Pull request by Ethroes**
Updated raw statistics. Built in one more hyperlink for FRC webpage. Corrected several grammar errors and made text less winded.

### CODE REVIEW

File: content/firstabout.html [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-together to help other teams when needed and to treat everyone with kindness and respect. Because of
-this, FIRST is said to not only teach students to build robots, but to build character as well.
+<a href="http://www.usfirst.org">FIRST (For Inspiration and Recognition of Science and Technology)</a> is a world-wide organization founded 
+by Dean Kamen in 1992. FISRT was designed to encourage students to pursue S.T.E.M. related careers 
+while instilling a sense of Gracious Professionalism. Gracious Professionalism is the mutual respect of oneself and others within 
+the heat of competition. FIRST is built around four levels of robotics competition. <a href="http://www.usfirst.org/roboticsprograms/jr.fll">Jr.FLL(grades K-3)</a>, 
+<a href="http://www.usfirst.org/roboticsprograms/fll">FLL(4-8)</a>, 
+<a href="http://www.usfirst.org/roboticsprograms/ftc">FTC(7-12)</a>, and 
+<a href="http://www.usfirst.org/roboticsprograms/frc">FRC(9-12)</a> enable students of all ages to 
+engage in FIRST.
```
Comment (murraybd) [inline content/firstabout.html:8]: "enable students of all ages to engage in FIRST" seems like the end of a sentence to me, but it starts off with Jr.FLL as far as I can tell. Am I missing something?

I also think there should be a space between the name and the grades e.g. "FLL (4-8)".

File: content/firstabout.html [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 <br><br>
 <img src="images/AboutFIRST2.JPG">
 <br><br>
 
-The FRC program reaches almost 60 thousand students across 12 different countries including Chile,
-Australia and Turkey. Each season these students compete against one another at various regional
-competitions across the United States and in their own countries. Each team is competing for a spot at the
-World Championship which is held in St. Louis, Missouri every April.
+Every year, FRC reaches over 78,000 students across 130 different competitions. These students make up the nearly
+3,100 teams that will compete for one of the 800 spots at the World Championships in St. Louis, Missouri.
```
Comment (murraybd) [inline content/firstabout.html:24]: I think it is still worth pointing out that FIRST is an international program, which was implied by mentioning 12 different countries.

**Comment by murraybd** [comment]
I have a couple of nit-picky comments and one bit (the enable students part) I don't understand.

**Comment by Ethroes** [comment]
Addressed highlighted issues

**Comment by murraybd** [comment]
Looks good, thanks!

**Pull request merged**